### PR TITLE
First draft of the other ways of support component

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

This is the first attempt at implementing a section of the bundle landing page in the new support project.

### About SVGs
Since our SVGs are not complicated, we decided to use a component with all the SVGs inline. Alternately, we could put all the SVGs in separated files, create a build step and use a webpack's SVG loader or file loader to load the SVG in the corresponding component. Nevertheless, considering our SVGs are simple (so far) we decided to go with the approach of the SVG component. Therefore, the developer whenever wants to add a new SVG, she or he must copy the content of the path tag inside the SVG component.    

[**Trello Card**](https://trello.com/c/oviKRuM6/483-other-ways-of-contribute-component)

## Changes

* Defined an SVG component with the first SVGs.
* Created the other ways of support component.
* Created Simple Header component.

## Future considerations and TODOs
* Define a way of using BEM style in shared components. 
* Import The Guardian's fonts.
* Define a class that handle the outer box of the page.

## Screenshots

picture 247

### CODE REVIEW

File: assets/components/ctaCircle/ctaCircle.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      fill: white;
+      height: 105%;
+      width: 105%;
+    }
+  }
+
+  span {
+    font-size: 14px;
+    line-height: 20px;
+    font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
```
Comment (Ap0c) [inline assets/components/ctaCircle/ctaCircle.scss]: `$gu-text-sans-web`

File: assets/components/svg/svg.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <svg viewBox={svgsCatalog[props.svgName].viewBox}>
+    <path d={svgsCatalog[props.svgName].path} />
+  </svg>
+);
+
+// ----- Proptypes ----- //
+
+svgs.defaultProps = {
+
+};
```
Comment (Ap0c) [inline assets/components/svg/svg.jsx]: ?

File: assets/pages/bundles-landing/bundlesLanding.jsx [hunk 3]
```diff
@@ -18,7 +19,11 @@ const store = createStore(reducer, 'PAPER+DIGITAL');
 
 const content = (
   <Provider store={store}>
-    <Bundles />
+    <div>
+      <Bundles />
+      <br style={{ clear: 'both' }} />
```
Comment (Ap0c) [inline assets/pages/bundles-landing/bundlesLanding.jsx]: Temporary addition?

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 4]
```diff
@@ -1,30 +1,131 @@
...
-	.bundles__bundle--paper {
-		background-color: #58a1ce;
-	}
+  .bundles__bundle--paper {
+      background-color: #58a1ce;
+  }
 
-	// ----- Shared Components ----- //
+  .ways-of-support {
+    max-width: 980px;
```
Comment (Ap0c) [inline assets/pages/bundles-landing/bundlesLanding.scss]: Does `map-get($mq-breakpoints, desktop)` work here?

Should we add another helper function to access the breakpoints independently?

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 5]
```diff
@@ -1,30 +1,131 @@
...
+
+      @include mq($from: desktop) {
+        width: 100%;
+      }
+    }
+  }
+
+  .ways-of-support__way {
+    background: gu-colour(comment-support-2);
+    color: guss-colour(neutral-1);
```
Comment (Ap0c) [inline assets/pages/bundles-landing/bundlesLanding.scss]: This doesn't exist any more 🙃

File: assets/pages/bundles-landing/bundlesLanding.jsx (hunk 3 above)
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.jsx]: Yes, we will define a formal way of separating sections of the page. Probably wrapping each section in `<section>` tags.

File: assets/components/svg/svg.jsx (hunk 2 above)
Comment (svillafe) [inline assets/components/svg/svg.jsx]: Added for the linter, but actually, it doesn't require it. It has been removed.

File: assets/pages/bundles-landing/bundlesLanding.scss (hunk 4 above)
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: It sounds fair, I will add that function to `breakpoint.css` and from now on, we will be able to use it.

**Pull request merged**